app/src/main/java/com/kickstarter/libs/Koala.java

```diff
@@ -180,26 +180,35 @@ public void trackActivityView(final int pageCount) {
 
   // SEARCH
   public void trackSearchView() {
-    client.track("Discover Search");
+    client.track(KoalaEvent.VIEWED_SEARCH);
+    client.track(KoalaEvent.DISCOVER_SEARCH_LEGACY);
```

Comment:
if you look around this file, you'll see examples of how we like to mark deprecated events with a

```java
// Deprecated event
```

comment to make it easier for removal later, would be great if you could add those comments in!